watertap/unit_models/zero_order/__init__.py

```diff
@@ -20,3 +20,5 @@
 from .coag_and_floc_zo import CoagulationFlocculationZO
 from .uv_zo import UVZO
 from .uv_aop_zo import UVAOPZO
+from .anion_exchange_zo import AnionExchangeZO
```

Comment:
Should this just be `ion_exchange`, with different subtypes for anion and cation exchange (and different resins in future)?

Reply:
I considered that but decided to take the route of separating them for now. I could look into cation_exchange and make sure it lines up with the anion_exhange unit in WT3.